### PR TITLE
[daint-gpu] Updating version of cray-python for Theano-0.9.0

### LINKED ISSUE #616: Theano build FAILED on daint and dom
The following `RegressionEB` build of `Theano` failed on `daint` and `dom`: 
```
Theano-0.9.0-CrayGNU-17.08-cuda-8.0-python3.eb
```

Link to the logs:
https://jenkins.cscs.ch/blue/organizations/jenkins/RegressionEB/detail/RegressionEB/771/pipeline/38

### PR DESCRIPTION
**Pull request by rsarm**
Fixes #616 

PyExtensions was added as a dependency since Theano's `setup.py` will download and install `six` if it can't  find it.

### CODE REVIEW

**Comment by rsarm** [comment]
retry this please

**Comment by lucamar** [comment]
`Dom` is under maintenance, therefore we will not be able to test the build there.

**Comment by lucamar** [comment]
retest this please

**Comment by lucamar** [comment]
retest this please

File: easybuild/easyconfigs/n/nose-parameterized/nose-parameterized-0.5.0-CrayGNU-17.08-python3.eb [hunk 1]
```diff
@@ -15,15 +19,15 @@ source_urls = [PYPI_SOURCE]
 sources = [SOURCE_TAR_GZ]
 
 dependencies = [
-             ('cray-python/17.06.1', EXTERNAL_MODULE),
+             ('cray-python/%s.1.1' % pyver, EXTERNAL_MODULE),
```
Comment (gppezzi) [inline easybuild/easyconfigs/n/nose-parameterized/nose-parameterized-0.5.0-CrayGNU-17.08-python3.eb]: I suggest to define the complete module version together with req_py_majver, req_py_majver and pyver and just use the new variable here (something like `py_mod_ver`)

File: easybuild/easyconfigs/n/nose-parameterized/nose-parameterized-0.5.0-CrayGNU-17.08-python3.eb [hunk 2]
```diff
@@ -15,15 +19,15 @@ source_urls = [PYPI_SOURCE]
...
 ]
 
 req_py_majver = 3
 req_py_minver = 5
```
Comment (gppezzi) [inline easybuild/easyconfigs/n/nose-parameterized/nose-parameterized-0.5.0-CrayGNU-17.08-python3.eb]: are these two variables still needed?

File: easybuild/easyconfigs/t/Theano/Theano-0.9.0-CrayGNU-17.08-cuda-8.0-python3.eb [hunk 3]
```diff
@@ -21,10 +23,11 @@ source_urls = ['https://pypi.python.org/packages/source/T/%s' % name]
 sources = [SOURCE_TAR_GZ]
 
 dependencies = [
-    ('cray-python/17.06.1', EXTERNAL_MODULE),
+    ('cray-python/%s.1.1' % pyver, EXTERNAL_MODULE),
```
Comment (gppezzi) [inline easybuild/easyconfigs/t/Theano/Theano-0.9.0-CrayGNU-17.08-cuda-8.0-python3.eb]: same change as for `nose-parameterized`

File: easybuild/easyconfigs/n/nose-parameterized/nose-parameterized-0.5.0-CrayGNU-17.08-python3.eb (hunk 2 above)
Comment (rsarm) [reply easybuild/easyconfigs/n/nose-parameterized/nose-parameterized-0.5.0-CrayGNU-17.08-python3.eb]: Yes. But here I have them twice, my mistake.

**Comment by gppezzi** [comment]
retest this please

**Pull request merged**